### DISCUSSION TITLE
ProjectTo issue

### DISCUSSION THREAD

**Discussion by amilcar-dev** [LuckyPennySoftware/AutoMapper#3669] [NA]
I'm getting the error "Argument types do not match" when using ProjectTo method to a dictionary. Although, when I use the same profile using the Map method it works fine. Any help?
Thanks

 CreateMap<Voucher, VoucherDto>()
                .ForMember(x => x.Descriptions, options => options.MapFrom(x =>
                    x.Product.Translations.Select(x =>
                        new KeyValuePair<Language, string>(x.Language, x.Description))));

**Comment by lbargaoanu** [comment]
Run that as a LINQ statement, without AM. You should get the same result.
